File: src/main/java/leets/leenk/domain/feed/domain/service/CommentSaveService.java
package leets.leenk.domain.feed.domain.service;

import leets.leenk.domain.feed.domain.entity.Comment;
import leets.leenk.domain.feed.domain.repository.CommentRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@Service
@RequiredArgsConstructor
public class CommentSaveService {
    private final CommentRepository commentRepository;

    public void saveComment(Comment comment) {
        commentRepository.save(comment);
    }
}

File: src/main/kotlin/leets/leenk/domain/feed/domain/service/CommentSaveService.kt
package leets.leenk.domain.feed.domain.service

import leets.leenk.domain.feed.domain.entity.Comment
import leets.leenk.domain.feed.domain.repository.CommentRepository
import org.springframework.stereotype.Service

@Service
class CommentSaveService(
    private val commentRepository: CommentRepository,
) {
    fun saveComment(comment: Comment) {
        commentRepository.save(comment)
    }
}
